plugin/src/main/java/com/denizenscript/denizen/utilities/blocks/FullBlockData.java

```diff
@@ -16,12 +16,17 @@
 import org.bukkit.block.data.*;
 import org.bukkit.block.data.type.RedstoneWire;
 import org.bukkit.block.data.type.Wall;
+import org.bukkit.block.data.type.Door;
+import org.bukkit.block.data.type.Stairs;
+import org.bukkit.block.data.type.Chest;
+import org.bukkit.block.data.Bisected;
 
 import java.lang.reflect.Method;
 import java.util.HashMap;
 import java.util.HashSet;
 import java.util.Map;
 
+

```

Comment:
stray newline

Reply:
Ah - should be cleaned up 904517d5d2512df75b60d8a11c46951697cb1f26